tests/ui/specs/Picker/Picker-specs.js

```diff
@@ -0,0 +1,102 @@
+const Page = require('./PickerPage');
+const {extractValue} = require('./Picker-utils.js');
+
+describe('Picker', function () {
+
+	describe('LTR locale', function () {
+		beforeEach(function () {
+			Page.open();
+		});
+
+		describe('default', function () {
+			const picker = Page.components.pickerDefault;
+
+			describe('5-way', function () {
+				it('should change the value forward when incrementing the picker', function () {
+					browser.waitUntil(() => picker.decrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightDown();
+					browser.waitUntil(() => picker.incrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightSelect();
+					browser.pause(500);
+					const newValue = extractValue(picker);
+					expect(newValue).to.equal('Korea');
+				});
+
+				it('should change the value backward when decrementing the picker', function () {
+					browser.waitUntil(() => picker.decrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightDown();
+					browser.waitUntil(() => picker.incrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightSelect();
+					Page.spotlightUp();
+					browser.waitUntil(() => picker.decrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightSelect();
+					browser.pause(500);
+					const newValue = extractValue(picker);
+					expect(newValue).to.equal('Romania');
+				});
+			});
+
+			describe('pointer', function () {
+				it('should increase the value when incrementing the picker', function () {
+					picker.incrementer(picker.self).click();
+					browser.pause(500);
+					const newValue = extractValue(picker);
+					expect(newValue).to.equal('Korea');
+				});
+
+				it('should decrease the value when decrementing the picker', function () {
+					picker.incrementer(picker.self).click();
+					browser.waitUntil(() => picker.incrementer(picker.self).isFocused(), {timeout: 1500,  interval: 100});
+					picker.decrementer(picker.self).click();
+					browser.pause(500);
+					const newValue = extractValue(picker);
+					expect(newValue).to.equal('Romania');
+				});
+			});
+		});
+
+		describe('with \'defaultValue\'', function () {
+			// supplied value is `Korea`
+			const picker = Page.components.pickerWithDefaultValue;
+
+			it('should have the default value selected', function () {
+				browser.pause(500);
+				const newValue = extractValue(picker);
+				expect(newValue).to.equal('Korea');
+			});
+		});
+
+		describe('disabled', function () {
+			const picker = Page.components.pickerDisabled;
+
+			describe('5-way', function () {
+				it('should not update on select', function () {
```

Comment:
Are we changing value by ok key?

Reply:
At the moment, this is how the Agate pickers work. Maybe we can have a discussion about how to make the component better for the 5-way navigation